src/Sylius/Bundle/ReviewBundle/Updater/ReviewableAverageRatingUpdater.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ReviewBundle\Updater;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Sylius\Component\Review\Calculator\AverageRatingCalculatorInterface;
+use Sylius\Component\Review\Model\ReviewableInterface;
+use Sylius\Component\Review\Model\ReviewInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.p.zalewski@gmail.com>
+ */
+class ReviewableAverageRatingUpdater implements ReviewableAverageRatingUpdaterInterface
```

Comment:
"Reviewable" is very confusing here, it suggests that the rating is reviewable. I'd simply say:

``AverageRatingUpdater`` or ``AverageReviewRatingUpdater``